src/jdk.crypto.cryptoki/share/native/libj2pkcs11/p11_convert.c

```diff
@@ -608,6 +614,67 @@ jTls12MasterKeyDeriveParamToCKTls12MasterKeyDeriveParamPtr(JNIEnv *env,
     return NULL;
 }
 
+/*
+ * converts the Java CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS object to a
+ * CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS pointer
+ *
+ * @param env - used to call JNI functions to get the Java classes and objects
+ * @param jParam - the Java CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS object to convert
+ * @param pLength - length of the allocated memory of the returned pointer
+ * @return pointer to the new CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS structure
+ */
+CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS_PTR
+jTlsExtendedMasterKeyDeriveParamToCKTlsExtendedMasterKeyDeriveParamPtr(JNIEnv *env,
+        jobject jParam, CK_ULONG *pLength)
+{
+    CK_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_PARAMS_PTR ckParamPtr;
+    jclass jTlsExtendedMasterKeyDeriveParamsClass;
+    jfieldID fieldID;
+    jlong prfHashMechanism;
+    jobject pSessionHash;
```

Comment:
Please initialize all locals (to `0` or `NULL`) where possible.

Reply:
See https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/760#discussion_r1534382772.